package.json

```diff
@@ -29,31 +27,44 @@
     "Google Cloud Key Management Service (KMS) API"
   ],
   "scripts": {
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "test": "c8 mocha",
-    "lint": "eslint '**/*.js'",
+    "test": "c8 mocha build/test",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "system-test": "mocha build/system-test",
+    "prelint": "gts fix && cd samples/ && npm install && cd .. && eslint --fix samples/*.js",
+    "fix": "gts fix",
     "docs": "jsdoc -c .jsdoc.js",
-    "system-test": "mocha system-test/ smoke-test/ --timeout 600000",
-    "fix": "eslint '**/*.js' --fix",
     "docs-test": "linkinator docs",
     "predocs-test": "npm run docs",
-    "prelint": "cd samples; npm link ../; npm i"
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r protos build/",
+    "compile-protos": "compileProtos src",
+    "prepare": "npm run compile",
+    "lint": "eslint samples/*.js"
```

Comment:
This needs `gts` too.  Please use https://github.com/googleapis/nodejs-vision/blob/master/package.json as a guide for how this should be set up throughout the run scripts